soc/riscv/riscv-privilege/telink_w91/idle.c

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
Why do we add this file? Are there any differences from standard risc-v implementation?

Reply:
removed